client/src/components/SudoModePasswordField/SudoModePasswordField.js

```diff
@@ -145,19 +173,38 @@ function SudoModePasswordField(props) {
     </div>;
   }
 
+  // Prevent a fout on initial render when rendering initiallyCollapsed
+  if (expanded === null) {
+    return null;
+  }
+
   // Render the component
   return <div className="sudo-mode-password-field">
-    <div className="sudo-mode-password-field-inner alert alert-info panel panel--padded">
+    { !expanded && <div className="sudo-mode-password-field__expander alert alert-info panel">
+      <div className="sudo-mode-password-field__expander-text-container">{ verifyMessageValue }</div>
+      <div className="sudo-mode-password-field__expander-button-container">
+        <Button
+          className="sudo-mode-password-field__expander-button font-icon-lock"
+          color="info"
+          onClick={() => handleExpand()}
+        >
+          { i18n._t('Admin.VERIFY', 'Verify') }
+        </Button>
+      </div>
+    </div>}
```

Comment:
This should probably be in a function, like the other `render<SomethingSpecific>` functions - though given how many review cycles this has had and that this doesn't affect the end result other than making it a little easier to visually parse the code, I won't hold off merging if this doesn't get done.

Reply:
Leaving as is